doc/nrf/app_dev/optimizing/power.rst

```diff
@@ -86,17 +86,75 @@ Cellular applications
 An nRF9160 DK can draw current ranging from a few micro amperes (in sleep mode) to hundreds of milli amperes (when the radio is active).
 To achieve long battery life, it is crucial that the application is optimized in the use of the radio.
 
+PSM usage
+=========
+
+Power Saving Mode (PSM) enables the modem to enter deep sleep when it does not need to communicate with the network.
+The modem can spend most of the time in deep sleep, only waking up when data needs to be sent and received.
+When implementing an application taking advantage of the PSM, keep in mind that the enabling PSM request does not mean that modem can always enter PSM.
```

Comment:
```suggestion
When implementing an application that uses PSM, keep in mind that the enabling PSM request does not mean that the modem can always enter PSM.
```

Reply:
Discussed offline with @trantanen that I'll change this to "requesting PSM".